src/components/SessionExpiredComponent.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import jwt from 'jsonwebtoken';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Button } from 'semantic-ui-react';
+import { expireSession, unexpireSession } from '../_actions/session.action';
+import ArtModal from './common/ModalComponent';
+
+
+export class SessionExpiredComponent extends React.Component {
+  componentDidMount() {
+    this.tokenValid();
+  }
+
+  componentDidUpdate() {
+    this.tokenValid();
+  }
+
+  tokenValid = () => {
+    const now = Date.now();
+    const jwToken = localStorage.getItem('art-prod-web-token');
+    if (jwToken) {
+      const token = jwt.decode(jwToken);
+      const { exp: tokenExpiry } = token || {};
+      if (now >= tokenExpiry * 1000) {
+        this.props.expireSession();
+      }
+    }
+  };
+
+  handleLogout = () => {
+    localStorage.removeItem('art-prod-web-token');
+    this.props.unexpireSession();
+    this.props.history.push('/');
+  };
+
+  render() {
+    return (
+      <div>
+        { this.props.sessionExpired &&
+        <ArtModal
+          trigger={null}
```

Comment:
Instead of adding this here, add it as a default prop in `ModalComponent.jsx`. I.e.

In `ModalComponent.jsx`
```
ArtModal.defaultProps = {
  ...
  trigger={null}
};

```
In fact, you don't need to add it as it is being [used as a ternary operator](https://github.com/AndelaOSP/art-dashboard/blob/develop/src/components/common/ModalComponent.jsx#L23) in the modal component. Only declare it as a default prop when eslint is complaining